Library/Homebrew/utils/git.rb

```diff
@@ -4,7 +4,7 @@ module Git
   module_function
 
   def last_revision_commit_of_file(repo, file, before_commit: nil)
-    args = [before_commit.nil? ? "--skip=1" : before_commit.split("..").first]
+    args = ["--skip=#{before_commit.nil? ? 1 : before_commit.split("..").first}"]
```

Comment:
The previous behaviour was wrong. If `before_commit` is not nil, it did not work correctly.